core/src/main/java/org/apache/iceberg/MetadataUpdateParser.java

```diff
@@ -462,6 +465,8 @@ private static MetadataUpdate readAddSchema(JsonNode node) {
     Schema schema = SchemaParser.fromJson(schemaNode);
     int lastColumnId;
     if (node.has(LAST_COLUMN_ID)) {
+      LOG.warn(
+          "Field last-column-id of MetadataUpdate.AddSchema is deprecated since Iceberg 1.8.0, will be removed in 1.9.0 or 2.0.0");
```

Comment:
Do we need to warn on every update? This is going to fill service logs. We should just ignore this.

Reply:
That works for me as well 👍